staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh

```diff
@@ -18,8 +18,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../../../../..
-EXAMPLE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=${GOPATH}/src/k8s.io/kubernetes
```

Comment:
Yes the intention was to make it easier for someone using the sample-apiserver as a base to modify the script and use it for their own purposes.